birb_log/src/lib.rs

```diff
@@ -0,0 +1,72 @@
+use birb::Module;
+use std::collections::HashMap;
+
+
+use std::time::SystemTime;
+
+
+
+pub enum LogCategory {
+    INFO,
+    DEBUG,
+    WARN,
+    ERROR
+}
+pub struct LogEntry {
+    pub module_str: String,
+    pub msg_str: String,
+    pub log_str: String,
```

Comment:
I'd rename these to "module", "message" and "log". Also I'd argue that the formatted string shouldn't be stored in the LogEntry and should be formatted when printed.